packages/react-ui/src/theme/carto-theme.js

```diff
@@ -1145,6 +1145,52 @@ export const cartoThemeOptions = {
         }
       }
     },
+
+    MuiTablePagination: {
```

Comment:
Thanks, I didn't know this file and now I understand better why styles behaved differently depending on where you looked at stuff :D